### ISSUE TITLE
Write tests for the api endpoints in go

### ISSUE THREAD

**Issue by freshteapot** [freshteapot/learnalist-api#9]
Figure out how to write the tests.

**Comment by freshteapot** [comment]
- This is on going and baby steps are happening each time I touch a file.

**Comment by freshteapot** [comment]
Current situation with test coverage once Issue #12 is closed, via PR #36.
```
github.com/freshteapot/learnalist-api/api/api/alist.go:15:		GetListsByMe			100.0%
github.com/freshteapot/learnalist-api/api/api/alist.go:29:		GetListByUUID			100.0%
github.com/freshteapot/learnalist-api/api/api/alist.go:50:		SaveAlist			96.8%
github.com/freshteapot/learnalist-api/api/api/alist.go:105:		RemoveAlist			87.5%
github.com/freshteapot/learnalist-api/api/api/api.go:31:		SetDomain			0.0%
github.com/freshteapot/learnalist-api/api/api/api.go:36:		Run				0.0%
github.com/freshteapot/learnalist-api/api/api/label.go:20:		PostUserLabel			75.0%
github.com/freshteapot/learnalist-api/api/api/label.go:49:		GetUserLabels			66.7%
github.com/freshteapot/learnalist-api/api/api/label.go:62:		RemoveUserLabel			81.8%
github.com/freshteapot/learnalist-api/api/api/root.go:9:		GetRoot				100.0%
github.com/freshteapot/learnalist-api/api/api/user.go:27:		PostRegister			100.0%
github.com/freshteapot/learnalist-api/api/api/version.go:16:		GetVersion			0.0%
```

**Comment by freshteapot** [comment]
Closing with the following info.

```
github.com/freshteapot/learnalist-api/api/api/alist.go:19:		V1GetListsByMe			100.0%
github.com/freshteapot/learnalist-api/api/api/alist.go:30:		V1GetListByUUID			100.0%
github.com/freshteapot/learnalist-api/api/api/alist.go:52:		V1SaveAlist			96.8%
github.com/freshteapot/learnalist-api/api/api/alist.go:112:		V1RemoveAlist			87.5%
github.com/freshteapot/learnalist-api/api/api/api.go:31:		SetDomain			0.0%
github.com/freshteapot/learnalist-api/api/api/api.go:36:		Run				0.0%
github.com/freshteapot/learnalist-api/api/api/label.go:20:		V1PostUserLabel			87.5%
github.com/freshteapot/learnalist-api/api/api/label.go:49:		V1GetUserLabels			66.7%
github.com/freshteapot/learnalist-api/api/api/label.go:62:		V1RemoveUserLabel		81.8%
github.com/freshteapot/learnalist-api/api/api/root.go:9:		V1GetRoot			100.0%
github.com/freshteapot/learnalist-api/api/api/user.go:27:		V1PostRegister			100.0%
github.com/freshteapot/learnalist-api/api/api/version.go:16:		V1GetVersion			100.0%
```
